Exposing Entitlement Management access package assignment approvals in v1.0 (#15132)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/entitlementmanagement-accesspackageassignmentapproval-stage-list.md

> ---

Reviewer: Fix file name to be **approval-list-stages.md** following the syntax: _parententityname-list-navpropertyname.md_

Reviewer 2: updating

> title: "List approvalStages"
> description: "List approval stages associated with an approval object."
> ms.localizationpriority: medium
> author: "sbounouh"
> ms.prod: "governance"
> doc_type: "apiPageType"
> ---
> 
> # List approvalStages

Reviewer: Title syntax is _List navPropertyName_
```suggestion
title: "List stages"
description: "List approval stages associated with an approval object."
ms.localizationpriority: medium
author: "markwahl-msft"
ms.prod: "governance"
doc_type: "apiPageType"
---

# List stages
```

Namespace: microsoft.graph

> In [Azure AD entitlement management](../resources/entitlementmanagement-overview.md), lists the [approvalStage](../resources/approvalstage.md) objects associated with an [approval](../resources/approval.md) object.  This call can be made by an approver, providing the identifier of the [access package assignment request](../resources/accesspackageassignmentrequest.md).

Reviewer: ```suggestion
In [Azure AD entitlement management](../resources/entitlementmanagement-overview.md), lists the [approvalStage](../resources/approvalstage.md) objects associated with an [approval](../resources/approval.md) object. This call can be made by an approver, providing the identifier of the [access package assignment request](../resources/accesspackageassignmentrequest.md).
```

## Permissions

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

| Permission type                        | Permissions (from least to most privileged) |
|:---------------------------------------|:--------------------------------------------|
| Delegated (work or school account)     | EntitlementManagement.Read.All, EntitlementManagement.ReadWrite.All |
| Delegated (personal Microsoft account) | Not supported. |
| Application                            | Not supported. |

## HTTP request

<!-- { "blockType": "ignored" } -->

```http

> GET /identityGovernance/entitlementManagement/accessPackageAssignmentApprovals/{id}/stages

Reviewer: ```suggestion
GET /identityGovernance/entitlementManagement/accessPackageAssignmentApprovals/{accessPackageAssignmentRequestId}/stages
```

```

## Request headers

| Name      |Description|
|:----------|:----------|
| Authorization | Bearer \{token\}. Required. |

## Request body

Do not supply a request body for this method.

## Response

> If successful, this method returns a `200 OK` response code and a collection of the [approvalStage](../resources/approvalstage.md) objects in the response body. However, if the caller does not have the right permissions, the method returns a `403 Forbidden` response code.

Reviewer: ```suggestion
If successful, this method returns a `200 OK` response code and a collection of the [approvalStage](../resources/approvalstage.md) objects in the response body. If the caller does not have the right permissions, the method returns a `403 Forbidden` response code.
```

## Examples

### Request

<!-- {
  "blockType": "request",

>   "name": "get_approval"

Reviewer: Unique name to avoid name clash with approval-get.md
```suggestion
  "name": "list_approvalStage"
```
